docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -163,11 +163,40 @@ The following command deletes the Alertmanager configuration in the Grafana Mimi
 mimirtool alertmanager delete
 ```
 
-#### Migrate Alertmanager configuration for UTF-8 in Alertmanager 0.27 and later
+#### Migrate Alertmanager configuration for UTF-8 in Mimir 2.12 and newer
 
-The migrate-utf8 command translates any matchers that are incompatible with the UTF-8 parser into
-equivalent matchers that are compatible by enforcing double quoting on the right hand side
-of the matcher. The translation does not change their behavior.
+This requires mimirtool version 2.12 or newer. You can check the version of mimirtool with
+`mimirtool version`.
+
+In accordance with [prometheus/prometheus#13095](https://github.com/prometheus/prometheus/issues/13095)
+and [prometheus/alertmanager#3486](https://github.com/prometheus/alertmanager/issues/3486), Mimir is
+adding support for UTF-8. To support UTF-8 in alerts, routes, silences and inhibition rules,
+Alertmanager has added a new parser for matchers that has a number of backwards incompatible changes.
+More information about these changes can be found [here](https://prometheus.io/docs/alerting/latest/configuration/#label-matchers).
```

Comment:
```suggestion
For more information about these changes, refer to [Label matchers](https://prometheus.io/docs/alerting/latest/configuration/#label-matchers) in the Prometheus documentation.
```

Reply:
We might want to remove the anchor and just link to https://prometheus.io/docs/alerting/latest/configuration/. If Prometheus updates this heading, our link could easily break.

Reply:
Ah. I think it's important to link to that specific section of the page. Is there some workaround we could do here?

Reply:
I just checked and we included `#label-matchers` in #8572.

Reply:
We can include the anchor if it's really important. Just something to be aware of.